gcloud/test_connection.py

```diff
@@ -255,7 +255,7 @@ def test_api_request_wo_json_expected(self):
             b'CONTENT',
         )
         self.assertEqual(conn.api_request('GET', '/', expect_json=False),
-                         b'CONTENT')
+                         u'CONTENT')
```

Comment:
@tseaver comment from f2a9c7dfa5f753fc007a8cea4741aa99131da5bb:

Per [RFC 1341](http://www.w3.org/Protocols/rfc1341/7_1_Text.html), we shouldn't decode `text/plain` without a `charset` as anything other than 'US-ASCII':

> The default character set, which must be assumed in the absence of a charset parameter, is US-ASCII. 

----

I'm unclear on how this applies? This relates to the change above with decoding to `utf-8` before the `expect_json` block.